worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
```

Comment:
I think `GetRaw()` must not be removed. Both `GetRaw()` and `GetLength()` should be the public `CompoundPacket` interface for `Transport` to get the raw content of the compound packet and send it.